cmd/agent/main.go

```diff
@@ -275,11 +275,16 @@ func buildProbe(logger *zap.SugaredLogger, probeJSON string) *readiness.Probe {
 		logger.Fatalw("Agent failed to parse readiness probe", zap.Error(err))
 		panic("Agent failed to parse readiness probe")
 	}
-	return readiness.NewProbe(coreProbe)
+	newProbe := readiness.NewProbe(coreProbe)
+	newProbe.InitialDelaySeconds = 10
```

Comment:
just curious the reason to add `InitialDelaySeconds` here

Reply:
There were errors in logs where it was aggressively probing, so wanted to give time for the service to be fully up...
The probe becomes successful on average after 10 seconds, that's why...

Reply:
This can be configured on the inference service yaml so might need to check if it is set then give this default value.